app/routes/farming-type.js

```diff
@@ -14,24 +16,7 @@ function createModel (errorMessage, data) {
           classes: 'govuk-fieldset__legend--l'
         }
       },
-      items: [
-
-        {
-          value: 'Crops for the food industry',
-          text: 'Crops for the food industry',
-          checked: !!data && (data.includes('Crops for the food industry'))
-        },
-        {
-          value: 'Horticulture',
-          text: 'Horticulture (including ornamentals)',
-          checked: !!data && (data.includes('Horticulture'))
-        },
-        {
-          value: 'Something else',
-          text: 'Something else',
-          checked: !!data && (data.includes('Something else'))
-        }
-      ],
+      items: setLabelData(data, ['Crops for the food industry', 'Horticulture (including ornamentals)', 'Something else']),
```

Comment:
this means that value of second  radiobutton is no longer 'Horticulture', but now 'Horticulture (including ornamentals)' - I think the value should be simple and remain 'Horticulture'

Reply:
@ogbk helper function updated to resolve your comments .